From 291d3f987abba18daea5a48a4fe39b4cd0a7dc3e Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 12 Jul 2023 00:39:42 +0100
Subject: [PATCH 1/1] data: Remove borked nvidia checks in udev rules file

We vendor enable these services in the /usr directory so is-enabled will
return disabled when they aren't symlinked to the /etc/ directory.

I guess this is a reason to use systemd presets.
---
 data/61-gdm.rules.in | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/data/61-gdm.rules.in b/data/61-gdm.rules.in
index a4f841bdf..631a5d375 100644
--- a/data/61-gdm.rules.in
+++ b/data/61-gdm.rules.in
@@ -53,12 +53,6 @@ TEST{0711}!="/usr/bin/nvidia-sleep.sh", GOTO="gdm_disable_wayland"
 TEST{0711}!="/usr/lib/systemd/system-sleep/nvidia", GOTO="gdm_disable_wayland"
 IMPORT{program}="/bin/sh -c \"sed -e 's/: /=/g' -e 's/\([^[:upper:]]\)\([[:upper:]]\)/\1_\2/g' -e 's/[[:lower:]]/\U&/g' -e 's/^/NVIDIA_/' /proc/driver/nvidia/params\""
 ENV{NVIDIA_PRESERVE_VIDEO_MEMORY_ALLOCATIONS}!="1", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_HIBERNATE=`systemctl is-enabled nvidia-hibernate`'"
-ENV{NVIDIA_HIBERNATE}!="enabled", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_RESUME=`systemctl is-enabled nvidia-resume`'"
-ENV{NVIDIA_RESUME}!="enabled", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_SUSPEND=`systemctl is-enabled nvidia-suspend`'"
-ENV{NVIDIA_SUSPEND}!="enabled", GOTO="gdm_disable_wayland"
 LABEL="gdm_nvidia_end"
 
 # If this machine has an internal panel, take note, since it's probably a laptop
-- 
2.40.1

